core/database_util.go

```diff
@@ -234,6 +238,67 @@ func GetBlock(db ethdb.Database, hash common.Hash) *types.Block {
 	return types.NewBlockWithHeader(header).WithBody(body.Transactions, body.Uncles)
 }
 
+// GetBlockReceipts retrieves the receipts generated by the transactions included
+// in a block given by its hash.
+func GetBlockReceipts(db ethdb.Database, hash common.Hash) types.Receipts {
+	data, _ := db.Get(append(blockReceiptsPrefix, hash[:]...))
```

Comment:
Not a blocker :) I just try to always make a case to avoid "data" as a variable because it's worst variable name there is :D 2nd worst is "data2". Followed by "value".

Even though all of these can be seen as binary blobs, they do contain different things. `receiptsRLP`, `txRLP` etc would be good name, informing the reader that the variable contains RLP of a certain Ethereum thing.